docs/learning/iModelHub/Briefcases.md

```diff
@@ -1,35 +0,0 @@
-# Briefcases
-
-[Briefcases](../Glossary.md#briefcase) are local copies of iModel that users can *acquire* to work with the iModel. Users can make changes to their copy of iModel and then [*push*](../Glossary.md#briefcase) them as a single [Changeset](../Glossary.md#changeset) file into iModelHub.
```

Comment:
I wonder if we need to keep part of this page, unless Briefcases are documented elsewhere too.